src/components/shared/basic/SearchResultItem/index.tsx

```diff
@@ -124,34 +146,44 @@ export const SearchResultItem = ({
         },
       }}
       onClick={() => {
+        console.log(item.category)
         switch (item.category) {
           case SearchCategory.PAGE:
+            dispatch(setAppear({ SEARCH: false }))
             navigate(`/${item.id}`)
             break
           case SearchCategory.OVERLAY:
             dispatch(show(item.id as OVERLAYS, ''))
             break
           case SearchCategory.ACTION:
+            dispatch(setAppear({ SEARCH: false }))
             dispatch(exec(item.id))
             break
           default:
             dispatch(show(getCategoryOverlay(item.category), item.id))
         }
       }}
     >
-      <ListItemIcon>{getIcon(item.category)}</ListItemIcon>
+      {file ? (
+        <ListItemIcon>
+          <img src={file} alt={item.category} />
```

Comment:
use from shared components:
`<Image src={'backend url of the image'} loader={fetchImageWithToken}>`